src/screens/Settings/MerchantAccountUtils.res

```diff
@@ -132,6 +132,20 @@ let getBusinessProfilePayload = (values: JSON.t) => {
     valuesDict->getString("three_ds_requestor_url", "")->getNonEmptyString,
   )
 
+  let outGoingWebHookCustomHttpHeaders = Dict.make()
+  let formValues = valuesDict->getDictfromDict("outgoing_webhook_custom_http_headers")
+
+  let _ =
```

Comment:
It won't return anything.
So we can remove let _ =

Reply:
It should've showned warning right

Reply:
yes